### DISCUSSION TITLE
看了下proxy的教程，原始的task在series没有结束的时候是不会被销毁的，对吗?

### DISCUSSION THREAD

**Discussion by chanchann** [sogou/workflow#657] [Q&A]
https://github.com/chanchann/workflow_annotation/blob/main/src_analysis/faq_62.md

**Comment by Barenboim** [comment]
原始task，也就是server task，和任何其他task一样在callback之后就不可用了。server task的callback时机是回复完成。
如果用户在这个callback里继续向series里插入任务，那么，series会继续执行，但server task已经不可使用了。至于内部什么时候真正delete对象，对用户来讲不重要。
